src/Projection/ProjectorId.php

```diff
@@ -13,7 +20,7 @@ final class ProjectorId
 {
     public function __construct(
         private readonly string $name,
-        private readonly int $version
+        private readonly int $version = 1
```

Comment:
Do we really want this default behaviour here? I think we should force the user to think about it and keep track of it, no?

Reply:
This is just a symptom of the problem i am having right now. In the documentation i noticed that it is difficult to explain what this ProjectorId is. And what a projection or projector is, or what the difference is.

I am going to revisit the naming and revisit the issue so that you only have to provide the ID if you are going to use it.